tmt/steps/provision/__init__.py

```diff
@@ -454,6 +459,241 @@ def requires(cls) -> List[str]:
         return []
 
 
+class ProvisionPlugin(tmt.steps.GuestlessPlugin):
```

Comment:
I don't understand the move of `ProvisionPlugin` and `Provision` classes, I thought I moved them to the necessary place in https://github.com/teemtee/tmt/pull/1439. The order should have been guest classes -> plugin class -> step class. Am I missing something, is there a reason to move these two classes in between `Guest` and `GuestSsh` classes?

Reply:
I believe commit 1b9e4367787960ffeac0804177fd31a7f1ec302a should be dropped. The class reordering in #1654 has been merged, so you can rebase on the latest `main`.